src/index.ts

```diff
@@ -290,10 +290,9 @@ prog
     if (!opts.noClean) {
       await cleanDistFolder();
     }
-    opts.name = opts.name || appPackageJson.name;
```

Comment:
ahhhh good catch, this line and the line below it were redundantly added in #130 , c.f. https://github.com/jaredpalmer/tsdx/pull/130#issuecomment-610716225 .

I'll remove both of these in a separate PR

Reply:
@agilgur5 Should I revert this fix?

Reply:
It might merge conflict with that PR, probably better to revert yea. I'm just about falling asleep so I won't make that PR today. Hopefully will get that in tomorrow and I'll make sure to give you credit there too (as `Co-Authored-By`) for the find

Reply:
OK! I will!